src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,97 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Interpretation for edges of GEOMETRY logical type, i.e. whether the edge
+ * between points represent a straight cartesian line or the shortest line on
+ * the sphere. It applies to all non-point geometry objects.
+ */
+enum Edges {
+  PLANAR = 0;
+  SPHERICAL = 1;
+}
+
+/**
+ * A custom binary-encoded polygon or multi-polygon to represent a covering of
+ * geometries. For example, it may be a bounding box or an envelope of geometries
+ * when a bounding box cannot be built (e.g. a geometry has spherical edges, or if
+ * an edge of geographic coordinates crosses the antimeridian). In addition, it can
+ * also be used to provide vendor-agnostic coverings like S2 or H3 grids.
+ */
+struct Covering {
+  /**
+   * A type of covering. Currently accepted values: "WKB".
+   */
+  1: required string kind;
+  /**
+   * A payload specific to kind. Below are the supported values:
+   * - WKB: well-known binary of a POLYGON or MULTI-POLYGON that completely
```

Comment:
```suggestion
   * - WKB: well-known binary [1] of a POLYGON or MULTI-POLYGON that completely
```

Not sure this formatting works, but this is the first mention of WKB. I originally just added it here
as I'm not sure that the name 'well-known binary' is actually that well known. Realized that there is the same link as below (currently line 325), but not sure if the [1] does auto-linking, or if this comment in the code should explain its own [1] link.

Reply:
Good suggestion! I moved definition of `GeometryEncoding` to the top.